Authenticator/Classes/OTPTokenListViewController.swift

```diff
@@ -152,8 +153,10 @@ extension OTPTokenListViewController /* UITableViewDataSource */ {
     }
 
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier(NSStringFromClass(TokenRowCell.self), forIndexPath: indexPath) as! TokenRowCell
-        updateCell(cell, forRowAtIndexPath: indexPath)
+        let cell = tableView.dequeueReusableCellWithIdentifier(NSStringFromClass(TokenRowCell.self), forIndexPath: indexPath)
+        if let cell = cell as? TokenRowCell {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 125 characters